tests/src/rules/prefer-default-export.js

```diff
@@ -129,4 +128,62 @@ ruleTester.run('prefer-default-export', rule, {
       }],
     }),
   ],
-})
+});
+
+context('Typescript', function() {
+  // Typescript
+  const parsers = ['typescript-eslint-parser', 'babel-eslint'];
+
+  if (semver.satisfies(eslintPkg.version, '>5.0.0')) {
```

Comment:
missing an `import semver from 'semver'` and `import eslintPkg from 'eslint/package.json'` at the top, i think?

Reply:
yup just updated, forgot these in my `git add -p`